config/initializers/dartsass.rb

```diff
@@ -0,0 +1,11 @@
+# config/initializers/dartsass.rb
+APP_STYLESHEETS = {
+  "application.scss" => "application.css",
+  "components/_result-card.scss" => "components/_result-card.css",
+  "components/_result-item.scss" => "components/_result-item.css",
+  "components/_result-sections.scss" => "components/_result-sections.css",
+  "visualise.scss" => "visualise.css",
+}.freeze
+
+all_stylesheets = APP_STYLESHEETS.merge(GovukPublishingComponents::Config.all_stylesheets)
+Rails.application.config.dartsass.builds = all_stylesheets
```

Comment:
This hasn't been documented yet (I'll make a note to include it in the documentation), but could you please add an additional line of configuration to silence dependency deprecation warnings like so:

```
Rails.application.config.dartsass.builds = all_stylesheets

Rails.application.config.dartsass.build_options << " --quiet-deps"
```
I initially thought that adding this line in the GOV.UK Publishing Components would suffice, but it appears that including it here is also necessary.